package.json

```diff
@@ -12,13 +12,14 @@
   "main": "lib/index.js",
   "scripts": {
     "lint": "eslint lib/ tests/",
```

Comment:
Not sure if this was left out on purpose or not...
Do we want to include `tools/` here too?

Reply:
ahh i missed it 👍